src/modules/emote_menu/stores/index.js

```diff
@@ -73,14 +73,23 @@ class EmoteStore extends SafeEventEmitter {
   }
 
   async updateEmoteProviders() {
-    const profilePicture = await twitch.getCurrentUserProfilePicture();
+    const [profilePicture, channelPicture] = await Promise.all([
+      twitch.getCurrentUserProfilePicture(),
+      twitch.getCurrentChannelProfilePicture(),
+    ]);
 
     emoteProviderCategories = [
       createCategory(
-        emotesCategoryIds.BETTERTTV,
-        'BetterTTV',
+        emotesCategoryIds.BETTERTTV_CHANNEL,
+        'BetterTTV Channel',
+        Icons.IMAGE(channelPicture == null ? cdn.url('/assets/logos/mascot.png') : channelPicture, 'BetterTTV'),
```

Comment:
it is confusing to use the same image for the channel all over the place. we need to have distinguishable icons

Reply:
any ideas on what the image could be instead?

Reply:
maybe there's some way we can build a component which renders the channel icon with a provider icon overlaid in the bottom right corner.